app/views/dev_mailer/pp.text.erb

```diff
@@ -0,0 +1 @@
+<%= @object.ai(plain:true, ruby19_syntax: true, indent: 2) %>
```

Comment:
I don't know what it stands for, but maybe `ascii inspect`:

![image](https://cloud.githubusercontent.com/assets/1001691/19744277/a8010dc6-9b81-11e6-83ff-28e9fa4034fd.png)

the ruby 1.9 thing is just the ruby 1.9 hash syntax, i.e. `x: 'blah'` instead of `x => 'blah'`.